### PR TITLE
Pri 702 breakerbox persist state

### PR DESCRIPTION
**Pull request by fatal-exception**

### CODE REVIEW

**Comment by fatal-exception** [comment]
For review only - please do not merge yet as kitchen tests are still running

File: templates/breakerbox.upstart.j2 [hunk 1]
```diff
@@ -13,9 +17,17 @@ end script
...
+                {% endif %}
                 --log-driver json-file \
                 --log-opt max-size=1g \
                 -p {{ breakerbox_docker_ports|join(' -p ') }} \
                 -v {{ breakerbox_docker_volumes|join(' -v ') }} \
                 breakerbox:{{ breakerbox_docker_version }} server /opt/conf/breakerbox.yml
 end script
+
+pre-stop script
+    /usr/bin/docker kill breakerbox-postgres || true
```
Comment (fatal-exception) [inline templates/breakerbox.upstart.j2]: oops! This should say 'breakerbox'

Comment (wunzeco) [reply templates/breakerbox.upstart.j2]: yes... I'll wait for the update

Comment (fatal-exception) [reply templates/breakerbox.upstart.j2]: corrected

Comment (fatal-exception) [reply templates/breakerbox.upstart.j2]: pushed again

**Review by wunzeco — APPROVED** [review]

**Comment by fatal-exception** [comment]
Thanks :)

**Pull request merged**